### PR TITLE
feat(levelpack): update long name / desc after insert

### PR DESCRIPTION
**Pull request by j-masci**
https://user-images.githubusercontent.com/24510397/112907497-f6460480-90bb-11eb-9d20-51984aa8af78.mp4

### CODE REVIEW

**Comment by j-masci** [comment]
closes this: https://github.com/elmadev/elmaonline-web/issues/19

File: src/components/FormResponse.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FormResponse = ({ msgs, showIfEmpty = false, isError }) => {
```
Comment (j-masci) [inline src/components/FormResponse.js:4]: I thought i saw on the front-end one time some html that seemed to do what this component does, but couldn't find anything in components folder. Do you know if anything? Just prints messages in an outlined box basically.

Comment (sunehs) [reply src/components/FormResponse.js:4]: Generally for this I've used https://github.com/elmadev/elmaonline-web/blob/dev/src/components/Feedback.js

**Review by sunehs — APPROVED** [review]

File: src/components/FormResponse.js (hunk 1 above)
Comment (j-masci) [reply src/components/FormResponse.js:4]: Oh ok. Could always re-factor to that in the future if it seems worthwhile. Would be more consistent, but also the new feature is a tiny one that not many people will even be likely to use IMO.

**Pull request merged**